projects/ngx-prx-styleguide/src/lib/upload/shared/audio.css

```diff
@@ -0,0 +1,87 @@
+.audio {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  background: #fff;
+  border-bottom: 1px solid #DDD;
+  border-top: 1px solid #DDD;
+  margin-top: -1px;
+  padding: 18px 22px;
+}
+.audio.changed {
+  box-shadow: inset 0px 0px 3px 3px #f09b4c;
+  -webkit-box-shadow: inset 0px 0px 3px 3px #f09b4c;
+  -moz-box-shadow: inset 0px 0px 3px 3px #f09b4c;
+  -o-box-shadow: inset 0px 0px 3px 3px #f09b4c;
+}
+.audio.invalid {
+  box-shadow: inset 0px 0px 3px 3px #e32;
+  -webkit-box-shadow: inset 0px 0px 3px 3px #e32;
+  -moz-box-shadow: inset 0px 0px 3px 3px #e32;
+  -o-box-shadow: inset 0px 0px 3px 3px #e32;
+}
+.audio.canceled {
+  animation: fade-out 0.5s 1;
+  -webkit-animation: fade-out 0.5s 1;
+  animation-fill-mode: forwards;
+  animation-delay: 0.5s;
+  -webkit-animation-delay: 0.5s;
+  -webkit-animation-fill-mode: forwards;
+}
+@keyframes fade-out { from { opacity :1; } to { opacity :0; } }
+@-webkit-keyframes fade-out { from { opacity :1; } to { opacity :0; } }
+
+.reorder {order: 1; flex-basis: 100%;}
+.cancel {order: 2; flex-basis: 100%;}
+.main {order: 3; flex-basis: 100%;}
+@media screen and (min-width:600px) {
+  .audio {flex-wrap: nowrap;}
+  .reorder {order: 1; flex-basis: 70px;}
+  .cancel {order: 3; flex-basis: 44px;}
+  .main {order: 2; flex-basis: 1;}
+}
+
+.reorder button {
+  font-size: 30px;
+  cursor: move;
+  margin: 0;
+}
+.cancel button {
+  margin: 4px 0 4px 20px
+}
+.audio.canceled .reorder button, .audio.canceled .cancel button {
+  display: none;
+}
+
+.main, .info {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: stretch;
+}
+.main .type {
+  border: 1px solid #DDDDDD;
```

Comment:
can we move over to the variables for colors? You just have to make it a `.scss` file